src/main/java/org/example/service/NotFoundService.java

```diff
@@ -0,0 +1,33 @@
+package org.example.service;
+
+public class NotFoundService {
+
+    private static NotFoundService INSTANCE;
+
+    public String getNotFoundResponse() {
+        StringBuilder stringBuilder = new StringBuilder();
+
+        stringBuilder
+                .append("<html>")
+                .append("<head>")
```

Comment:
Read about header tag. What is the main responsibility of this tag?

Reply:
fixed